README.md

```diff
@@ -19,6 +19,12 @@ This extension provides the following commands that can be keybound in your [key
 - `terminalTabs.showTerminal9`: Shows the ninth terminal
 - `terminalTabs.showTerminal10`: Shows the tenth terminal
 
+__Note__: some common user-preferred shortcuts - eg, `Ctrl T` - are normally sent to terminal itself, rather than to the outside environment (where things like opening new terminal tabs are done). In order for that short cut to be used, open *File*, *Preferences*, and add the following to your user settings (on the right):
+
+    "terminal.integrated.commandsToSkipShell": [
+      "terminalTabs.createTerminal"
```

Comment:
This would actually break all the other terminal keybindings as the commandsToSkipShell will replace the previous value. For now this should be something like:

```js
"terminal.integrated.commandsToSkipShell": [
  ...defaults...,
  "terminalTabs.createTerminal"
]
```

Reply:
This is no longer true.